Algebraic_foundations/include/CGAL/number_utils.h

```diff
@@ -324,6 +324,36 @@ NT approximate_sqrt(const NT& nt)
   return approximate_sqrt(nt, Sqrt());
 }
 
+template <class NT>
+Comparison_result
```

Comment:
I wonder if it should return something a bit more generic, so that when NT is Interval_nt it would return `Uncertain<Comparison_result>`. The way it is used currently, it probably wouldn't matter much.

Reply:
Done in https://github.com/CGAL/cgal/pull/5532/commits/17b254639059bd247700d7c8c7e9fce3ff72dc42.

Reply:
Didn't you write that the .h file must also be included?  And how is this change related to the fact that we want to get `Do_intersect_3` division-free?

Reply:
> Didn't you write that the .h file must also be included?

You are right: I forgot. I ran the CTest testsuite, though, so it is probably already included. I will add it anyway for completeness.

>  And how is this change related to the fact that we want to get `Do_intersect_3` division-free?

That change is only a reaction to Marc's comment: https://github.com/CGAL/cgal/pull/5532#discussion_r607908030